app-misc/qmodmaster/qmodmaster-0.5.2.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="A free Qt-based implementation of a ModBus master application"
+DESCRIPTION="A graphical user interface allows easy communication with ModBus \
+ RTU and TCP slaves.
+
+-Features
+     ModBus RTU/TCP Master
+     Monitor Bus activity [ Raw Data and PDU details ]
+"
```

Comment:
This looks better in HaikuDepot (last double-qoute should be at the end of the last line):
```diff
--Features
-     ModBus RTU/TCP Master
-     Monitor Bus activity [ Raw Data and PDU details ]
-"
+Features
+* ModBus RTU/TCP Master
+* Monitor Bus activity [ Raw Data and PDU details ]"
```

Reply:
ok